doobie/src/test/scala/za/co/absa/fadb/doobie/AvailableMetaInstancesTest.scala

```diff
@@ -18,85 +18,86 @@ package za.co.absa.fadb.doobie
 
 import doobie.util.meta.Meta
 import org.scalatest.funsuite.AnyFunSuite
+import za.co.absa.fadb.tags.IntegrationTestTag
 
 class AvailableMetaInstancesTest extends AnyFunSuite {
 
-  test("Meta[java.time.OffsetDateTime] available from doobie.implicits.javatimedrivernative") {
+  test("Meta[java.time.OffsetDateTime] available from doobie.implicits.javatimedrivernative", IntegrationTestTag) {
```

Comment:
I don't think any of these are integration tests. BTW, if they would be all ITs, why not to set it on class level, instead of individual tests level?

Reply:
You are correct. Removed Integration test mark.